src/EnergyPlus/Coils/CoilCoolingDXCurveFitOperatingMode.cc

```diff
@@ -290,6 +290,17 @@ void CoilCoolingDXCurveFitOperatingMode::CalcOperatingMode(EnergyPlus::EnergyPlu
     // Currently speedNum is 1-based, while this->speeds are zero-based
     auto &thisspeed(this->speeds[max(speedNum - 1, 0)]);
 
+    if ((inletNode.MassFlowRate == 0.0) || ((PLR == 0.0) & (speedNum == 1)) || ((speedNum == 0) && (speedRatio == 0))) {
```

Comment:
Does this mean just a little but true? ((PLR == 0.0) & (speedNum == 1))

Reply:
That' line's been changed a bit over [here](https://github.com/NREL/EnergyPlus/pull/9250) but, yeah, I'm not sure what the compiler did with a single & instead of &&.

Reply:
The expressions evaluate to `bool & bool`, which becomes a bitwise AND on the 2 bools.  Bools are like integer 1s and 0s, so that's why it can work, but of course it should be the full && here.